elixir/test/sqlite_vss_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule SqliteVssTest do
+  use ExUnit.Case
+  doctest SqliteVss
+
+  test "greets the world" do
+    assert SqliteVss.hello() == :world
```

Comment:
Could you add a test here to check that `SqliteVss.loadable_path_vss0()` returns a string/path here? No need to test it with a sqlite DB connection, unless you find it easy to do